tests/index.js

```diff
@@ -40,7 +41,8 @@ const compare = async (template, path, result) => {
 			console.log(color.red(`\n\nMismatch in ${template}. The following diff was generated:`));
 
 			const diff = dmp.diff_main(expectation, result, false);
-			for (let i = 0; i < diff.length; i++) {
+			for (let i = 0; i < diff.length; i += 1) {
+				// eslint-disable-next-line prefer-const
```

Comment:
You could add `all` to the eslintrc: https://eslint.org/docs/rules/prefer-const#destructuring